client/src/components/MarketUpdate/TurnipForm.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from "react";
+import { getCurrentDateTime, getDays } from "./TurnipFormSelectors";
+
+export default (props) => {
+	const { handleSubmit } = props;
+
+	const defaultDateTime = getCurrentDateTime();
+	const times = getDays();
+
+	const options = [];
+	for (const day in times) {
+		options.push(
+			<option key={times[day].getDay()} value={times[day]}>
+				{day}
+			</option>
+		);
+	}
+
+	const [date, setDay] = useState(defaultDateTime.date);
+	const [timeOfDay, setTimeOfDay] = useState(defaultDateTime.timeOfDay);
+	const [price, setprice] = useState(undefined);
+
+	return (
+		<section>
+			<form
+				onSubmit={(event) => {
+					handleSubmit(event, { date, timeOfDay, price });
+				}}
+			>
+				<select
```

Comment:
I think we might need a label or some sort of ARIA tag for accessibility. If you have a quick fix for now, cool, but we can come back to it. I'm realizing there are a few things I've pushed that need to be gone back over to improve accessibility as well 😕